runtime/lib_ksils12.c

```diff
@@ -859,7 +948,8 @@ rsksiCtxOpenFile(rsksictx ctx, unsigned char *logfn)
 	}
 
 done:
-	ctx->ksi = ksi;
+	/* Register ksi file in rsksictx for keeping track of block timeouts. */
+	rsksiRegisterKsiFile(ctx, ksi);
```

Comment:
no return error verification

Reply:
This should be fixed during another task where overall error handling is improved (and fixed).